lib/dialects/db2/index.js

```diff
@@ -1,14 +1,47 @@
 'use strict';
 
 const _ = require('lodash');
-const AbstractDialect = require('../abstract');
+const { AbstractDialect } = require('../abstract');
 const ConnectionManager = require('./connection-manager');
 const Query = require('./query');
 const QueryGenerator = require('./query-generator');
 const DataTypes = require('../../data-types').db2;
 const { Db2QueryInterface } = require('./query-interface');
 
 class Db2Dialect extends AbstractDialect {
+  static supports = _.merge(_.cloneDeep(AbstractDialect.supports), {
```

Comment:
Can we only set the ones that are different from AbstractDialect here?

Reply:
Also counts for the other dialects of course

Reply:
Done. And updated the typings accordingly